ui/apps/platform/src/Containers/Policies/Modal/ImportPolicyJSONModalError.tsx

```diff
@@ -66,12 +68,14 @@ function ImportPolicyJSONError({
         >
             <>
                 <ModalBoxBody>
-                    Address the errors below to continue importing policies
+                    {duplicateErrorsOnly
+                        ? 'Address the errors below to continue importing policies'
+                        : 'Correct the errors that are listed, and then try to import again.'}
```

Comment:
i don't think i understand the difference between the two messages? especially given we were already checking for duplicate errors before and only showed a single error message

Reply:
The first one shows as a header to a form where you can choose between ("Rename policy" OR "Keep both policies") AND "Overwrite existing policy".

For the latter, since there are multiple policies each with an error, it's meant to convey fix the errors and try again. No options to automatically fix for you.